### PR TITLE
Add csp-preflight and csp-setup for Azure BYOA support

### PR DESCRIPTION
**Pull request by valeriodelsarto**

### CODE REVIEW

File: azure/biganimal-csp-setup [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+  echo "Options:"
+  echo "  -d, --display-name: The name of Azure AD App."
+  echo "  -s, --subscription: The Azure Subscription ID used by BigAnimal."
+  echo "  -y, --years:        [Optional] The Number of years for which the credentials will be valid. Only accept positive integer value. Default: 1 year."
+  echo "  -i, --id:           [Optional] The Application (client) ID of Service Principal will be updated."
+  echo "  -h, --help:         Show this help."
+  echo ""
+}
+
+check()
```
Comment (YanniHu1996) [inline azure/biganimal-csp-setup:72]: like `aws/biganimal-csp-setup`, can we output some messages which tell users the `setup` script is running?

Comment (valeriodelsarto) [reply azure/biganimal-csp-setup:72]: Sure! I'll add this part too!

Comment (valeriodelsarto) [reply azure/biganimal-csp-setup:72]: Added here https://github.com/EnterpriseDB/cloud-utilities/pull/49/files#diff-4a15b2ca5ecfc13050cbfef0857fc00659a5143e708f6a540bef7245de8a50f3R76-R90

**Review by YanniHu1996 — APPROVED** [review]
LGTM

**Pull request merged**